extension/scripts/features/avg-personal-stat/avg-personal-stat.entry.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
+
+(async () => {
+	if (!getPageStatus().access) return;
+
+	featureManager.registerFeature(
+		"Average Personal Stat",
+		"personalstats",
+		() => settings.pages.profile.avgpersonalstats,
+		init,
+		init,
+		null,
+		null,
+		async () => {
+			await checkDevice();
+		}
+	);
+
+	function init() {
+		addFetchListener((event) => {
+			const {
+				page,
+				json,
+				fetch: { body },
+			} = event.detail;
+			if (page !== "personalstats") return;
+			if (body.includes("getGraphData")) {
+				const graphData = json;
+				calculateStatsAverage(graphData);
+			}
+		});
+	}
+
+	function calculateStatsAverage(graphData) {
+		for (let stat in graphData.data) {
+			const statData = graphData.data[stat];

```

Comment:
You could use a `for (const ... of ...)` loop here.

Reply:
can you clarify what you mean here?

Reply:
As with the `const` rule for variables that will not be modified later, `for` loop variables must also be declared `const`.